Vanja M. Karan name update.

anthology-assist opened the pull request:
https://aclanthology.org/2020.coling-main.559.pdf is hard to edit, left unchanged
https://aclanthology.org/L12-1470/ do not know where the PDF is http://www.lrec-conf.org/proceedings/lrec2012/pdf/796_Paper.pdf

mjpost reviewed (changes requested):
A few notes:
* Names should be centered better here: https://preview.aclanthology.org/vanja-name-update/2024.lrec-main.413.pdf
* It seems commas are missing: https://preview.aclanthology.org/vanja-name-update/2021.socialnlp-1.12.pdf
* Missed this one: https://preview.aclanthology.org/vanja-name-update/2020.coling-main.559.pdf
* Missed: https://preview.aclanthology.org/vanja-name-update/people/vanja-m-karan
* Missed: https://preview.aclanthology.org/vanja-name-update/W19-3707.pdf
* Font mismatch, also missing commas: https://preview.aclanthology.org/vanja-name-update/W18-4422.pdf
* Missed:

anthology-assist commented:
@mjpost 

> A few notes:
> 
> * Names should be centered better here: https://preview.aclanthology.org/vanja-name-update/2024.lrec-main.413.pdf

Addressed

> * It seems commas are missing: https://preview.aclanthology.org/vanja-name-update/2021.socialnlp-1.12.pdf

Adding commas completely messes up the PDF format (creating a new author line)
> * Missed this one: https://preview.aclanthology.org/vanja-name-update/2020.coling-main.559.pdf

Addressed
> * Missed: https://preview.aclanthology.org/vanja-name-update/people/vanja-m-karan

Not sure which one you are referring to here

> * Missed: https://preview.aclanthology.org/vanja-name-update/W19-3707.pdf

Addressed
> * Font mismatch, also missing commas: https://preview.aclanthology.org/vanja-name-update/W18-4422.pdf

Can't add commas, it messes up the author line

> * Missed:

Not sure which one you are referring to here

This one is missing http://www.lrec-conf.org/proceedings/lrec2012/pdf/796_Paper.pdf because I'm not sure where the PDF is.
